shared/common-adapters/list2.native.tsx

```diff
@@ -17,7 +17,7 @@ class List2<T> extends PureComponent<Props<T>> {
   _getItemLayout = (data, index) => {
     switch (this.props.itemHeight.type) {
       case 'fixed':
-        return {height: this.props.itemHeight.height, index, offset: this.props.itemHeight.height * index}
+        return {length: this.props.itemHeight.height, index, offset: this.props.itemHeight.height * index}
```

Comment:
seems odd?

Reply:
yeah, this was the bug where I thought react-native was buggy. It's always been there and the reason why it didn't break other stuff was we just never used the `fixed` version of `List2`